src/internal_events/loki.rs

```diff
@@ -32,40 +41,76 @@ impl InternalEvent for LokiUniqueStream {
 }
 
 #[derive(Debug)]
-pub struct LokiOutOfOrderEventDropped;
+pub struct LokiOutOfOrderEventDroppedError {
+    pub count: usize,
+}
 
-impl InternalEvent for LokiOutOfOrderEventDropped {
+impl InternalEvent for LokiOutOfOrderEventDroppedError {
     fn emit_logs(&self) {
-        debug!(
+        error!(
```

Comment:
@spencergilbert I think you'd recently changed this to be `debug` because it was confusing users?

https://github.com/vectordotdev/vector/issues/10401

Reply:
Well then, for the `Dropped` either 🤔 Nothing should be considered as an error in this file

Reply:
It does seem like every but the unlabeled event are not really errors. I don't believe we have firm documentation around the these not errors however